pkg/openshift/filesystem/filesystem.go

```diff
@@ -39,17 +47,22 @@ func NewFilesystem(name string) (Filesystem, error) {
 	return &filesystem{name}, nil
 }
 
-func (f *filesystem) mkdirAll(name string, perm os.FileMode) error {
-	return os.MkdirAll(name, perm)
+func (f *filesystem) Mkdir(name string, fileInfo Fileinfo) error {
+	return os.Mkdir(name, fileInfo.Mode)
```

Comment:
add a comment to this and MkdirAll: note this doesn't chown/chgrp name as would require root permissions

Reply:
added.

Reply:
please add the comment to Mkdir as well